meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc

```diff
@@ -23,4 +24,5 @@ do_install() {
         -C ${S} \
         DESTDIR=${D} \
         install
+    ln -sf ${datadir}/mender-monitor/mender-monitord ${D}${bindir}/mender-monitord
```

Comment:
Still pending discussion for this in https://github.com/mendersoftware/monitor-client/pull/17#discussion_r664089103

Reply:
Discussion there is done, decided to keep the link.

However this is already done by the `install` target, so we should not need it here.

Reply:
discussed over gmeet. thanks!